### ISSUE TITLE
A user can progress to the Auditee Information step

### ISSUE THREAD

**Issue by Jkrzy** [GSA-TTS/FAC#226]
## Background

As an auditee, I want to be able to access the `Auditee Information` page  so that I can provide the information necessary begin the SF-SAC. 

## Accept Criteria: 

**Given** that I entered my information on the first step, [Submission Criteria Check form](https://federalist-9a617ff8-042d-4076-9581-bb999f9c6639.app.cloud.gov/site/gsa-tts/fac-frontend/audit/new/step-1/), and was determined eligible
**When** I click Continue
**Then** I’m taken to the next step in the submission process, [the Auditee Info page](https://federalist-9a617ff8-042d-4076-9581-bb999f9c6639.app.cloud.gov/site/gsa-tts/fac-frontend/audit/new/step-2/), where I can provide my UEI and FY information. 

## Not in scope
* We don't need to provide a direct link to the Auditee Info Page, it should only be accessible by completing the previous step.

## Remaining tasks
- https://github.com/GSA-TTS/FAC/issues/322

**Comment by petermarks-gsa** [comment]
Blocked on https://github.com/GSA-TTS/FAC/issues/235

**Comment by petermarks-gsa** [comment]
@jasnakai Per Slack discussion today, sounds like you are still working on this and it's not yet ready for review. Moving back to in progress.

**Comment by petermarks-gsa** [comment]
Verified end to end by @timoballard and @jasnakai via django admin (see screen). That info gets stored on the user in the backend until the 3rd form is submitted at which point the info gets written to the database.
![image](https://user-images.githubusercontent.com/100610443/178057707-0068fa77-3bdc-473c-86b3-39cc9dc5ede4.png)
